### ISSUE TITLE
SDDM doesn't respect mouse speed settings from KDE settings

### ISSUE THREAD

**Issue by mthw0** [sddm/sddm#1252]
For example: I have a high DPI mouse and so I lower the speed so the cursor doesn't move too fast. This setting is not applied to SDDM and thus it's a lot faster. I would say that people expect their mouse to behave the same across the entire system, which is why I think this is an issue.

**Comment by plfiorini** [comment]
Each user has different Plasma settings, which one should be used? And why should SDDM read Plasma settings at all, why not GNOME settings?

**Comment by mthw0** [comment]
You have a good point. How about using settings from the last used session? Or there is an option in SDDM-KCM (the settings module for SDDM in KDE) to synchronize settings between KDE and SDDM that currently include: color settings, fonts, cursor theme, etc. Would it be possible to also "synchronize" mouse speed settings?

**Comment by flipwise** [comment]
Yes, it's on us (KDE) and a specific user to sync ourselves with SDDM, not the other way around.

I've had a quick look and I'm currently not seeing any way to set pointer speed in SDDM so I guess the ability to do so needs to be added here first (and the issue rephrased).

We would then be able to sync it in via sddm-kcm.

**Comment by Pointedstick** [comment]
Yeah it's not so much that SDDM should read Plasma settings, but that SDDM should allow setting a cursor speed property (like the DPI property), which desktop environments would then write to in the config file as needed.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/sddm/sddm/issues/1252#issuecomment-622472275

@Pointedstick, I believe that it would be better if SDDM could read the configuration of the currently-selected user. Windows does that via its lock-screen to ensure accessibility support. Without that, it ultimately remains inconsistent, and unusable for those with motor impairments using shared computer systems.

**Comment by GERJanB** [comment]
Is there any plan to implement this? Or has it already been implemented somewhere else as I am facing the same issue right now.
